src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -1066,13 +1067,21 @@ class EventView {
 
       if (item.value === DisplayModes.TOP5 || item.value === DisplayModes.DAILYTOP5) {
         if (this.getAggregateFields().length === 0) {
-          return {...item, disabled: true};
+          return {
+            ...item,
+            disabled: true,
+            tooltip: t('At least one aggregate function is required to use this view.'),
```

Comment:
I think while aggregate function is more accurate, might want to stick with just `function` to match the column builder:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/4205004/113060761-15f23100-917f-11eb-9ec5-5b34c3c82e29.png">

Or maybe, add a function that groups events?